plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeAnalyze.java

```diff
@@ -765,11 +778,224 @@ public void testCollectStatsAfterColumnAdded(boolean collectOnWrite)
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testNoStats()
+            throws IOException, URISyntaxException
+    {
+        String tableName = copyResourcesAndRegisterTable("no_stats");
+        assertQuery(
+                "SELECT * FROM " + tableName,
+                """
+                        VALUES
+                        (42, 'foo'),
+                        (12, 'ab'),
+                        (null, null),
+                        (15, 'cd'),
+                        (15, 'bar')
+                        """);
+
```

Comment:
Let's showcase here that the table has indeed no stats whatsoever:
- no `extended_stats.json`
- no file level stats

```
        assertQuery(
                "SHOW STATS FOR " + tableName,
                """
                        VALUES
                        ('c_int', null, null, null, null, null, null),
                        ('c_str', null, null, null, null, null, null),
                        (null, null, null, null, null, null, null)
                        """);
```